server/src/main/java/io/seata/server/coordinator/DefaultCore.java

```diff
@@ -269,10 +269,6 @@ public boolean doGlobalCommit(GlobalSession globalSession, boolean retrying) thr
         // if it succeeds and there is no branch, retrying=true is the asynchronous state when retrying. EndCommitted is
         // executed to improve concurrency performance, and the global transaction ends..
         if (success && globalSession.getBranchSessions().isEmpty()) {
-            if (!retrying) {
```

Comment:
Why are these changes involved?

Reply:
In order to be aggregated into sessionHelper for unified processing